src/main/resources/solr/setup/core-collection.sh

```diff
@@ -6,59 +6,4 @@ if [ $# -eq 0 ]; then
 fi
 
 # Set specific field types and multiValued field properties in Solr schema
-curl -X POST -H 'Content-type:application/json' --data-binary '{
-  "add-field":{
-    "name":"authors",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":true
-  },
-  "add-field":{
-    "name":"contributors",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":true
-  },
-  "add-field":{
-    "name":"identifiers",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":true
-  },
-  "add-field":{
-    "name":"journals",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":false
-  },
-  "add-field":{
-    "name":"relations",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":true
-  },
-  "add-field":{
-    "name":"subjects",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":true
-  },
-  "add-field":{
-    "name":"topics",
-    "type":"string",
-    "stored":true,
-    "multiValued":true,
-    "docValues":true
-  },
-  "add-field":{
-    "name":"datePublished",
-    "type":"string",
-    "stored":true
-  }
-}' http://localhost:8983/solr/"$1"/schema
+curl -X POST -H 'Content-type:application/json' --data-binary '@src/main/resources/solr/schemas/core-collection.json' http://localhost:8983/solr/"$1"/schema
```

Comment:
Can probably make this script take some more parameters (schema file name, Solr collection name which already exists) to make this generic

Reply:
Yea, I was gonna but thought we don't need more generic right now (`yagni` lol). I'll probably add, feels weird to not have generic